Instruments/source/BTagEffEstimator.cxx

```diff
@@ -74,11 +75,15 @@ class BTagEffEstimator {
         std::string channel_all = "all";
         std::set<std::string> channel_names = channels;
         channel_names.insert(channel_all);
-        if(args.deep_csv()== "yes"){
+        if(args.csv_type() == "deepCSV"){
```

Comment:
you can use BTagger class also here to make the code more flexible for future changes.